### DISCUSSION TITLE
Takie tam ...

### DISCUSSION THREAD

**Discussion by AroKol78** [MajkiIT/polish-ads-filter#21037] [Show and tell]
~`https://github.com/FiltersHeroes/KAD/discussions`~
`https://github.com/MajkiIT/polish-ads-filter/discussions`
można po naszemu?
<details><summary>pic:</summary>

![proszepoPL](https://user-images.githubusercontent.com/70756447/170484513-fdb3c580-54a3-4fb8-88d5-98fe9e7aeb8a.PNG)

</details>

---
tak se przeglądam
`https://majkiit.github.io/polish-ads-filter/docs/` -> `https://github.com/qarmin/Instrukcje-i-Tutoriale/blob/master/FirefoxBezpieczenstwo.md#readme`

`security.tls.version.min = 2 - Skuteczniejsze szyfrowanie TLS 1.2 (może powodować problemy ze stronami które nie wspierają silnego szyfrowania tls 1.2)`
według mojej wiedzy: `2` = TLS1.1, `3` = TLS1.2 (możliwe, że są inne kwiatki)

---
github albo ja źle liczę, ~chodzi o moje ulubione/gwiazdki (`Stars` :star:), niby mam 60 a pokazuje 59~
już nie aktualne, zniknęło z mojej listy jedno repo "ManchildProductions/MyPal" (`github.com/community/community/discussions/17193`)

zauważyłem, że github ma problemy z liczeniem w wielu miejscach
<details><summary>pic:</summary>

![licz01](https://user-images.githubusercontent.com/70756447/172836972-300ab062-4f48-4834-951d-28be98176611.PNG)

</details>

**Comment by AroKol78** [comment]
mam problem na stronie `geekweek.interia.pl` z osadzonym wideo z yt np.
`https://geekweek.interia.pl/styl-zycia/ciekawostki/news-nasa-ma-poczucie-humoru-w-kosmos-poleci-maskotka-stara-sruba,nId,6201608`
<details><summary>pic:</summary>

![gwibug001](https://user-images.githubusercontent.com/70756447/183089130-013e9233-b61f-479b-bf13-c3917e801ed9.JPG)
-

dodałem `geekweek.interia.pl##.col-lg-4.col-md-4.hidden-sm.hidden-rs.hidden-xs.main-sidebar`
jest lepiej, a i tak jakby obraz rozciągnięty, da się z tym coś zrobić? (wideo 1519x495?, mam podziałkę 1440x900)
![gwibug002](https://user-images.githubusercontent.com/70756447/183089152-995de67b-442d-4cb4-8157-eac7c50d0c63.JPG)
![gwi003](https://user-images.githubusercontent.com/70756447/183096583-eff5c0ca-4f47-40f7-8c56-2242d410374f.PNG)

</details>

**Comment by krystian3w** [comment]
Nie wiem nic o błędzie na Interii, spolszczenia raczej majkiit i xxcriticxx nie dodadzą.

W PAF jest jako takie: https://github.com/FiltersHeroes/PolishAnnoyanceFilters/discussions (być może moje z Google), więc potrzebujemy tłumacza by jak najmniej stracił się sens

![obraz](https://user-images.githubusercontent.com/35370833/188515298-26b536f4-2e8b-43de-924c-999018cdef76.png) | Google?!
--- | ---


Też uważam, że z tym wpisem do about:config to jest jakiś fail.

1 komentarz a 0 widocznych to może spam był (a użytkownik ma (shadow)banana 🍌)? 🪦

**Comment by AroKol78** [comment]
zawrotu głowy można dostać 😄 w wielu miejscach na `forum.dobreprogramy.pl`
nigdzie indziej nie spotkałem z czymś takim, część obrazków dobrze się wyświetla a część to jakiś kołowrotek
może to tylko u mnie (Serpent52-2022.09.09)

<details><summary>przykład:</summary>

`https://forum.dobreprogramy.pl/t/pop-up-reakcji-na-telefonie-jest-popsuty/656873`
`techforum.wpcdn.pl/optimized/3X/b/1/b[...]0.png` , `techforum.wpcdn.pl/original/3X/7/9/7[...]c.png`
![cosnietak001](https://user-images.githubusercontent.com/70756447/189892010-d11376b5-2987-4205-921b-7cb0e6f37ec0.JPG)

</details>

**Reply by krystian3w** [reply]
Serwer nie wyrabia po aktualizacji silnika forum.

**Reply by AroKol78** [reply]
<details><summary>c.d.:</summary>

`https://forum.dobreprogramy.pl/t/konto-ms-online-usuwanie-kluczy-bitlocker/655574/6`
> Screny padły po aktualizacji forum:

![dziwne003](https://user-images.githubusercontent.com/70756447/190146931-4630f070-a83c-465a-81f8-ad53da3dcb92.JPG)
![dziwne002](https://user-images.githubusercontent.com/70756447/190145679-7d72d15f-4b20-407b-93f3-2691ea391a18.JPG)

</details>

**Reply by krystian3w** [reply]
To był jakiś błąd osadzana wideo z Facebooka.

**Reply by AroKol78** [reply]
zauważyłem, że tam gdzie mam kręcioła z obrazkiem jest w konsoli
<details><summary>błąd:</summary>

`Warning: Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://techforum.wpcdn.pl/original/[...].png. (Reason: missing token ‘credentials’ in CORS header ‘Access-Control-Allow-Headers’ from CORS preflight channel).`
`Error: TypeError: NetworkError when attempting to fetch resource.`

</details>

czy to coś poważnego, czy to błąd przeglądarki czy może błąd u nich?

**Reply by hawkeye116477** [reply]
To jest błąd u nich.

**Comment by AroKol78** [comment]
czy ktoś zauważył, że są problemy z załadowaniem `https://easylist.to/easylist/easylist.txt` (analogicznie z `easyprivacy`)(u mnie już drugi dzień)
sama strona od razu się ładuje i niema problemu `https://easylist.to/`

**Reply by hawkeye116477** [reply]
Takie problemy to raczej nie tutaj, tylko do Fanboya...

**Reply by AroKol78** [reply]
przez taki problem listy dłużej się aktualizują na uBO - zanim przemieli i dojdzie do lustra
(`easylist/easylist/issues/13389`)

**Reply by AroKol78** [reply]
jak ktoś jest niecierpliwy (tak jak ja) i nie chce czekać na naprawę tego problemu to może wyłączyć `easylist` i `easyprivacy` w uBO,
a zapodać listę "2in1" `https://easylist-downloads.adblockplus.org/easyprivacy+easylist.txt`

**Comment by AroKol78** [comment]
Cud 😄 👍  się stał na stronach należących do wupe - przeglądarka PM lub pochodne.
